app/Annotations/OpenApi/controllersAnnotations/modalityAnnotations/AnnotationsModality.php

```diff
@@ -1,24 +1,52 @@
-<?php 
+<?php
+
 namespace App\Annotations\OpenApi\controllersAnnotations\modalityAnnotations;
 
+use OpenApi\Annotations as OA;
+
 class AnnotationsModality
 {
-
-     /**
-     * @OA\Get(
-     *     path="/api/v1/modality",
-     *     summary="Retrieve a specialization list from resume model enum",
-     *     tags={"Modality"},
-     *     @OA\Response(
-     *         response=200,
-     *         description="Successful. Specialization list retrieved from enum",
-     *         @OA\JsonContent(
-     *             type="array",
-     *             @OA\Items(type="string")
-     *         )
-     *     )
-     * )
-     */
-
+    /**
+ * @OA\Get(
+ *     path="/modality/{studentId}",
```

Comment:
If you change the url of your endpoint make sure to notify the frontend people who are adding this endpoint.

Reply:
Should I have created an issue to refactor the names?

Reply:
Probably. This is outside this issue scope.